ReplicationCommands.md

```diff
@@ -0,0 +1,80 @@
+# Replication Work
```

Comment:
This should be in an issue so it is kept as history when/if this file is ever deleted.

Reply:
Totally - opened https://github.com/dataplat/dbatools/issues/8995 and I'll remove this file from the repo - thanks!